app/src/main/java/org/fossasia/phimpme/gallery/activities/SingleMediaActivity.java

```diff
@@ -385,6 +393,13 @@ public void onPageChanged(int oldPosition, int position) {
             adapter = new ImageAdapter(LFMainActivity.listAll, basicCallBack, this, this);
             getSupportActionBar().setTitle(all_photo_pos + 1 + " " + getString(R.string.of) + " " + size_all);
             current_image_pos = all_photo_pos;
+            if (bottomMenu.findItem(R.id.action_favourites).getIcon() == null) {
```

Comment:
Are you sure of using this logic? Shouldnt be `.getColorFilter()`?

Reply:
getIcon() is returning the Drawable object so if drawable is null there should not any chance that getColorFilter() gets called , it will throw a NullPointerException to invoke a method on null object.